pkg/envoy/rbac/policy.go

```diff
@@ -47,7 +47,7 @@ func (p *Policy) Generate() (*xds_rbac.Policy, error) {
 			for _, orPrincipalRule := range principalRuleList.OrRules {
 				// Fill in the authenticated principal types
 				if orPrincipalRule.Attribute == DownstreamAuthPrincipal {
-					authPrincipal := getPrincipalAuthenticated(orPrincipalRule.Value)
+					authPrincipal := GetPrincipalAuthenticated(orPrincipalRule.Value)
```

Comment:
Would `GetAuthenticatedPrincipal` read better?  🤔

Reply:
I'll consider renaming these helpers in a separate PR.